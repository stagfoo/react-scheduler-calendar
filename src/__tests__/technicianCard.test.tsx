import { shallow } from 'enzyme';
import React from 'react';
import '../setupTest';
import TechnicianCard from '../technicianCard';

describe('technician card', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<TechnicianCard name={'name'}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
