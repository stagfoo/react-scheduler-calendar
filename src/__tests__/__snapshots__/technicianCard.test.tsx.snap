// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`technician card should render correctly 1`] = `
<div>
  <ForwardRef(UserOutlined) />
  name
</div>
`;
